src/intl/en/page-developers-docs.json

```diff
@@ -1,64 +1,88 @@
 {
   "docs-nav-accounts": "Accounts",
+  "docs-nav-accounts-descp": "Entities in the network that can hold a balance and send transactions",
```

Comment:
Same as above - `descp` -> `description` or `desc`

(We actually flip flop between these two @wackerow - do we have a preference?)

Reply:
I have changed _descp_ to _description_.